docs/synapse/userguides/syn_tools_feed.ipynb

```diff
@@ -11,7 +11,235 @@
     "Synapse Tools - feed\n",
     "====================\n",
     "\n",
-    "TBD"
+    "The Synapse ``feed`` tool is a way to ingest data exported from one Cortex into another Cortex. Users should be familiar with both the Synapse data model (:ref:`data-model-terms` et al.) as well as the Synapse concepts such as packed nodes and splices in order to use and understand the ``feed`` tool effectively.\n",
```

Comment:
"....as well as the Synapse concepts such as...." -> "...as well as Synapse concepts..."